src/components/combobox/combobox.tsx

```diff
@@ -419,14 +423,16 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
         }
         break;
       case "Home":
-        event.preventDefault();
-        this.active = true;
+        this.active ? event.preventDefault() : null;
```

Comment:
Can you use `if` statements instead of using conditionals? The latter is more commonly used for assigning values. This applies to other lines in this method.